Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -925,6 +927,7 @@ extension BrowserViewController: WKNavigationDelegate {
                 urlBar.updateURL(tab.displayURL);
                 toolbar?.updateBackStatus(webView.canGoBack)
                 toolbar?.updateFowardStatus(webView.canGoForward)
+                toolbar?.updateBrowserToolbarButtonsStatus(webView.URL!)
```

Comment:
Rather than creating a new callback to handle this, we should just use the existing methods on BrowserToolbar (`updateBookmarkStatus` and `updateReloadStatus`). The browser toolbar shouldn't care about the current URL, so by having BrowserViewController own the disable logic and update the toolbar, we maintain the separation of concerns. We'll need to add `updateShareStatus` to the toolbar since that doesn't exist yet.